runtime/Cargo.toml

```diff
@@ -53,6 +53,7 @@ pallet-sudo = { git = "https://github.com/paritytech/substrate", default-feature
 pallet-timestamp = { git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.13" }
 pallet-transaction-payment = { git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.13" }
 pallet-transaction-payment-rpc-runtime-api = { git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.13" }
+pallet-collective = { git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.13" }
```

Comment:
I thought we didn't need this any more

Reply:
I don't see any other way to get around [this issue](https://github.com/substrate-developer-hub/substrate-parachain-template/issues/87) that's preventing us from running benchmarks for quite a while.